style.css

```diff
@@ -0,0 +1,311 @@
+@import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600&display=swap');
+@import url('https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css');
+
+/*General Styling*/
+:root{
+
+    /*color codes*/
+    --instagram-blue: #0595F6;
+    --light-blue: #0F4173;
+    --dark-blue: #00376B;
+    --light-grey: #FAFAFA;
+    --dark-grey: #A1A1A1;
+    --mild-grey: #DBDBDB;
+}
+
+html {
+    font-size: 62.5%;
+}
+
+body{
+    font-family: 'Open Sans', sans-serif;
+    font-size: 16px;
```

Comment:
We still have some usages of `px` in this file, please replace them with the corresponding `rem` values